languages/purescript/exercises/concept/basics/.docs/after.md

```diff
@@ -0,0 +1,5 @@
+In the introduction, we explained functions and said that every function has one or more arguments. This is not strictly true. Technically all functions have only one argument; this is called [function currying][function-currying]).
```

Comment:
Does PureScript have partial application? If so, it might be useful to explain this fact in that exercise. But feel free to leave it in here. I like the video!

```suggestion
In the introduction, we explained functions and said that every function has one or more arguments. This is not strictly true. Technically all functions have only one argument; this is called [function currying][function-currying].
```

Reply:
Yes, these are all identical (both their types and the generated code):
```purs
fn x y z = ...
fn = \x y z -> ...
fn = \x -> \y -> \z -> ...
```